### DISCUSSION TITLE
Landscape mode in Android Reversing Text

### DISCUSSION THREAD

**Discussion by stealthworks** [Blazored/Modal#313] [General]
HI
I have an odd issue in that when I am in a Blazored modal Popup and rotate the screen to Landscape and enter values into a text field, when I then rotate the screen back to portrait, all the text in the field is reversed. For example, if I enter 123 into a text field in landscape and then rotate the screen to portrait , the field now contains 321 !!! This is very weird and only seems to happen on Android, not on an iPhone! Has anyone seen this issue before?

I'm posting to this channel as I only see it when using a Blazored modal so, was wondering if this is a known issue with this component or not?

Thanks in advance for any help

**Comment by larsk2009** [comment]
This is not something I have heard before, and I don't see how the Modal could be causing this. Yet, never say never, maybe some weird thing is causing this. If you can share a project with us which reproduces this issue, we might be able to check what is causing it.

**Reply by stealthworks** [reply]
Thank you for getting back to me so quickly. I'll upload a project over the weekend for you to take a look at
Many thanks :)
